meshtastic/interfaces/ble/interface.py

```
@@ -25,18 +26,18 @@
 import threading
 import time
 from queue import Empty
 from threading import Event
-from typing import (
-    IO,
-    Any,
-    Awaitable,
-    Callable,
-    TypeVar,
-)
+from typing import IO, Any, Awaitable, Callable, TypeVar, cast
```

Ruff (0.15.4)

[warning] Import from `collections.abc` instead: `Awaitable`, `Callable`

Import from `collections.abc`

(UP035)